libs/src/utils/apiSigning.js

```diff
@@ -24,13 +24,13 @@ function recoverWallet (web3, response) {
 }
 
 /**
- * Sort the object keys alphabetically
- * @param {object} x
+ * Recursively sorts object keys alphabetically
  */
-function _sortKeys (x) {
+function sortObjectKeys (x) {
```

Comment:
this name change is safe as this function was previously not exported and not referenced anywhere else